node/src/accountant/db_big_integer/big_int_db_processor.rs

```diff
@@ -253,28 +276,33 @@ impl<'a, T: TableNameDAO> BigIntSqlConfig<'a, T> {
     }
 }
 
-//to be able to display things that implement ToSql
-pub trait ExtendedParamsMarker: ToSql + Display {}
+// This is a native Rusqlite format for data supply to their interface
+type RusqliteParamPair<'params> = (&'params str, &'params dyn ToSql);
+// This is a representation of the above, plus providing a way to display the values (not possible natively)
+type DisplayableRusqliteParamPair<'params> = (&'params str, &'params dyn DisplayableParamValue);
```

Comment:
Since, we can't define what the first and second item of the tuple represent in `RusqliteParamPair` because it's part of the native format. I'd suggest, if we improve that by providing field names to the `DisplayableRusqliteParamPair` by changing it to struct.